server/src/main/java/io/mantisrx/master/jobcluster/JobClusterHelper.java

```diff
@@ -0,0 +1,86 @@
+package io.mantisrx.master.jobcluster;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import io.mantisrx.common.Label;
+import io.mantisrx.runtime.command.InvalidJobException;
+import io.mantisrx.server.master.domain.JobDefinition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class JobClusterHelper {
```

Comment:
should this be called LabelManager? Dedicated class to take care of all the system labels, resolving labels on job submits etc.

Reply:
Yeah initially I was thinking a general purpose class for misc. methods but yeah a dedicated class makes sense at the moment.